build.go

```diff
@@ -61,6 +61,13 @@ func Build(entries EntryResolver, dependencies DependencyManager, planRefinery B
 
 		logger.SelectedDependency(entry, dependency, clock.Now())
 
+		source, _ := entry.Metadata["version-source"].(string)
+		if source == "buildpack.yml" {
+			logger.Subprocess("WARNING: Setting the MRI version through buildpack.yml will be deprecated soon in MRI Buildpack v0.1.0.")
+			logger.Subprocess("Please specify the MRI version through an environment variable configurtion. See README.md for how to do this.")
+			logger.Break()
+		}
+
```

Comment:
Looking for feedback if there's a better place to have this warning message/ a more concise message to put out.

Reply:
Two thoughts:
1. I think the version that will trigger the deprecation is `v1.0.0`.
2. Since there is only a single configuration option, it may make sense to just tell the user what the environment variable is, instead of telling them to go read docs.